client-react/src/App.tsx

```diff
@@ -1,12 +1,29 @@
+import classNames from "classnames/bind";
+import {
+  BrowserRouter,
+  Switch,
+  Route,
+} from "react-router-dom";
 import styles from './App.module.scss'
 import { Layout } from './features/layout/Layout';
+import { TodoDetailsPage } from "./features/todos/components/TodoDetailsPage/TodoDetailsPage";
 import { TodoListContainer } from './features/todos/components/TodoListContainer/TodoListContainer';
+import { TodoListPage } from './features/todos/components/TodoListPage/TodoListPage';
+
+const classes = classNames.bind(styles)
 
 function App() {
+  const appStyles = classes({
+    app: true
+  })
+
   return (
-    <div className={styles['App']}>
+    <div className={appStyles}>
       <Layout>
-        <TodoListContainer containerOptions={{hasActions: true}}/>
+        <Switch>
+          <Route exact path="/" component={TodoListPage} />
+          <Route path="/details/:id" component={TodoDetailsPage} />
```

Comment:
Path name is not explanatory. What's details is this? Todo details? Can we make it like todos/details/id?

Add a route /todos and reroute from route '/' to 'todos' automatically everytime

Reply:
Fixed